packages/block-library/src/image/view.js

```diff
@@ -23,10 +23,23 @@ const { state, actions, callbacks } = store(
 	'core/image',
 	{
 		state: {
-			currentImageId: null,
+			isGallery: false,
```

Comment:
This could be derived (maybe in the future).

```suggestion
			get isGallery() {
        return state.images.length > 1;
      },
```

Reply:
renamed `isGallery` to `hasNavigation` in 1bf3f27a53adf605c8a556abcfedc6939eb96b54, and it is a getter based on number of images.

Reply:
This also means when there is only one image in the gallery (or only one image enabled with lightbox), left/right icons are hidden instead of being disabled.

Reply:
And think that's better. Actually, @paaljoachim mentioned here that this should be the default behavior on the first and last images.